libraries/AP_Camera/AP_Camera_Backend.h

```diff
@@ -125,6 +128,13 @@ class AP_Camera_Backend
         uint32_t feedback_trigger_logged_count; // ID sequence number
     } camera_feedback;
 
+    // Picture settings
+    struct {
+        bool enabled;               // true if taking multiple pictures is enabled
+        float time_interval;        // time interval (in seconds) between two consecutive pictures
+        uint32_t num_remaining;     // number of pictures still to be taken
```

Comment:
Let's make num_remaining a uint16_t.  Surely 65k pictures is enough..

Reply:
I thought in future we may be taking lakhs of pictures :laughing: